lib/modules/auth/login/bloc/login_state.dart

```diff
@@ -0,0 +1,29 @@
+import '/models/login/login_response_model.dart';
+
+abstract class LoginState {
+  const LoginState();
+}
+
+class LoginInitialState extends LoginState {
+  const LoginInitialState();
+}
+
+class LoginLoadingState extends LoginState {
+  const LoginLoadingState();
+}
+
+class LoginSuccessState extends LoginState {
+  final LoginResponseModel model;
+
+  const LoginSuccessState(this.model);
+}
+
+class LoginErrorState extends LoginState {
```

Comment:
you use only on State Error state and change the message according to each error ?!!!